frontend/shared/PulsingIcon.svelte

```diff
@@ -130,8 +130,8 @@
   }
 
   .gradio-webrtc-pulsing-icon-container:has(.icon-image.full-screen) {
-    width: 300px;
-    height: 300px;
+    width: 100%;
```

Comment:
If the icon the developer passes in is really big, would it get sized appropriately?

Reply:
yes the `gradio-webrtc-icon-wrapper` limits the max-height to 128px